src/main/java/org/opentripplanner/transit/raptor/rangeraptor/internalapi/Heuristics.java

```diff
@@ -45,11 +45,6 @@ public interface Heuristics {
    */
   int bestOverallJourneyTravelDuration();
 
-  /**
-   * Return the best/minimum required number of transfers from origin to destination.
-   */
-  int bestOverallJourneyNumOfTransfers();
```

Comment:
More in doubt about this one, but we will start improving the performance on OTP soon, so I would leave it until then.